src/Config.php

```diff
@@ -27,6 +27,11 @@ public function getRules(): array
     {
         return [
             '@PSR2' => true,
+            'array_syntax' => ['syntax' => 'short'],
+            'php_unit_method_casing' => false,
+            'trailing_comma_in_multiline_array' => true,
+            'no_trailing_comma_in_singleline_array' => true,
+            'no_unused_imports' => true,
```

Comment:
@LucWollants is going to love this! 😄 👌

Reply:
Is there also a check & automatic fix for making sure imports are sorted alphabetically?

Reply:
Yes, it's part of the PSR-12 ruleset so we're already enforcing that.

Reply:
So much ❤️ in this PR 😬